apps/web/components/molecules/opinionItem/opinionItem.tsx

```diff
@@ -0,0 +1,56 @@
+import { Card, Heading, Text } from 'ui';
+import Image from 'next/image';
+import clsx from 'clsx';
+import type { Opinion } from '../../../types/types';
+
+type OpinionItemProps = Opinion;
+
+export const OpinionItem = ({
+	author,
+	avatar,
+	rating,
+	comment,
+}: OpinionItemProps) => {
+	return (
+		<div className="p-2">
+			<Card tag="div">
+				<div className="flex flex-col md:flex-row justify-center items-center">
+					<Image
+						src={avatar}
+						alt=""
+						width={1000}
+						height={1000}
+						className="w-full h-full rounded-full w-24 h-24 object-cover"
```

Comment:
Also add `aria-hidden="true"` to be 100% sure

Reply:
For what purpose?